WalletWasabi.Fluent/ViewModels/WalletManagerViewModel.cs

```diff
@@ -110,6 +110,12 @@ public WalletManagerViewModel()
 
 	public ObservableCollection<WalletViewModelBase> Wallets => _wallets;
 
+	public bool TryGetSelectedWalletViewModel([NotNullWhen(true)] out WalletViewModel? walletViewModel)
```

Comment:
The name is misleading as it only returns the selected wallet if that is logged in.

Reply:
`TryGetSelectedAndLoggedInWalletViewModel` ?

Reply:
Feels long but don't have a better idea, ack.

Reply:
Done